.github/workflows/macOS-build.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  push:
+    branches: [ "master", "stage" ]
+  pull_request:
+    branches: [ "master", "stage" ]
+
+jobs:
+  build:
+
+    runs-on: macos-10.15
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+        submodules: recursive
+    - name: Build
+      run: |
+        brew install openssl@1.1
+
+        megaWheel="macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.macosx_10_11_intel.macosx_10_11_x86_64.macosx_10_12_intel.macosx_10_12_x86_64.whl"
+        baseWheelName="macosx_10_11_x86_64"
+        sslLibPath="/usr/local/Cellar/openssl@1.1/1.1.1l/lib/"
+        cClientVersion=""
+
+        export STATIC_SSL=1
+        export SSL_LIB_PATH=$sslLibPath
+        export CPATH=/usr/local/Cellar/openssl@1.1/1.1.1l/include/
+        pushd .
+
+        python3 setup.py sdist --formats=gztar
```

Comment:
Ram originally wrote: "this sdist (source code dict) should be done only one time, not for all python flavor."

Isn't there only one Python flavor (CPython)? Do you mean versions? Each Python version is used for a separate job.

Reply:
Aerospike is built for various python versions and for source tar, no need to do for every version